src/main/java/duke/ui/DialogBox.java

```diff
@@ -0,0 +1,64 @@
+package duke.ui;
+
+import java.io.IOException;
```

Comment:
I believe that the import statement for 'java.io.IOException' should be in the 'STANDARD_JAVA_PACKAGE' group.
Though I could be wrong.